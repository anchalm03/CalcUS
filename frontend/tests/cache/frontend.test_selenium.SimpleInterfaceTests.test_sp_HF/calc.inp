TITLE "File created by ccinput"
start in
memory total 2000 mb
charge 0

geometry units angstroms noautosym
H   -1.43860000  -0.26820000   0.00000000
H   -2.03860000  -0.26820000   0.00000000
end

basis
* library Def2-SVP
end

scf
singlet
end

task scf energy

