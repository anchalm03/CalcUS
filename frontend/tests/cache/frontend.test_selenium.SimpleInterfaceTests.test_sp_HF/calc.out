 argument  1 = calc.inp
                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.0.2
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2020
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = web
    program         = nwchem
    date            = Mon Jul  3 14:25:36 2023

    compiled        = Wed_Jan_06_22:30:30_2021
    source          = /build/nwchem-vBXbhr/nwchem-7.0.2
    nwchem branch   = 7.0.2
    nwchem revision = N/A
    ga revision     = 5.7.1
    use scalapack   = T
    input           = calc.inp
    prefix          = in.
    data base       = ./in.db
    status          = startup
    nproc           =       16
    time left       =     -1s



           Memory information
           ------------------

    heap     =   65535998 doubles =    500.0 Mbytes
    stack    =   65535995 doubles =    500.0 Mbytes
    global   =  131072000 doubles =   1000.0 Mbytes (distinct from heap & stack)
    total    =  262143993 doubles =   2000.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                              File created by ccinput
                              -----------------------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)


          ------
          auto-z
          ------
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 H                    1.0000     0.30000000     0.00000000     0.00000000
    2 H                    1.0000    -0.30000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       0.8819620817

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.60000


            XYZ format geometry
            -------------------
     2
 geometry
 H                     0.30000000     0.00000000     0.00000000
 H                    -0.30000000     0.00000000     0.00000000

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 H                |   1 H                |     1.13384  |     0.60000
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          1
 ==============================================================================





 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          Def2-SVP                  on all atoms 


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.30107010E+01  0.019682
  1 S  1.96225720E+00  0.137965
  1 S  4.44537960E-01  0.478319

  2 S  1.21949620E-01  1.000000

  3 P  8.00000000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                          Def2-SVP                 3        5   2s1p


                                 NWChem SCF Module
                                 -----------------


                              File created by ccinput



  ao basis        = "ao basis"
  functions       =    10
  atoms           =     2
  closed shells   =     1
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./in.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                          Def2-SVP                 3        5   2s1p



 Forming initial guess at       0.1s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -0.99855801

      Non-variational initial energy
      ------------------------------

 Total energy =      -0.839218
 1-e energy   =      -2.372956
 2-e energy   =       0.651776
 HOMO         =      -0.658611
 LUMO         =       0.171435


 Starting SCF solution at       0.1s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


 #quartets = 2.310D+02 #integrals = 5.870D+02 #direct =  0.0% #cached =100.0%


 Integral file          = ./in.aoints.00
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  44630
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1       -1.1053911951  1.21D-01  1.18D-01      0.1
                 2       -1.1073017598  2.13D-02  2.13D-02      0.1
                 3       -1.1073666012  4.25D-05  4.23D-05      0.1


       Final RHF  results 
       ------------------ 

         Total SCF energy =     -1.107366601241
      One-electron energy =     -2.700984882452
      Two-electron energy =      0.711656199544
 Nuclear repulsion energy =      0.881962081666

        Time for solution =      0.1s


             Final eigenvalues
             -----------------

              1      
    1   -0.6388
    2    0.2226
    3    0.4453
    4    1.0953
    5    1.4414
    6    1.4414
    7    2.2382
    8    2.3304
    9    2.3304
   10    4.9636

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    1  Occ=2.000000D+00  E=-6.388362D-01
              MO Center= -3.6D-17,  1.9D-19, -1.3D-18, r^2= 6.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.420892  1 H  s                  6      0.420892  2 H  s          

 Vector    2  Occ=0.000000D+00  E= 2.226173D-01
              MO Center=  1.0D-15, -6.7D-21,  4.6D-20, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      2.459536  1 H  s                  7     -2.459536  2 H  s          

 Vector    3  Occ=0.000000D+00  E= 4.452878D-01
              MO Center= -5.1D-16,  2.8D-18,  3.2D-17, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.721318  1 H  s                  7      0.721318  2 H  s          
     1     -0.654705  1 H  s                  6     -0.654705  2 H  s          

 Vector    4  Occ=0.000000D+00  E= 1.095305D+00
              MO Center=  2.6D-16, -1.7D-20,  1.1D-19, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.685929  1 H  s                  7     -1.685929  2 H  s          
     1     -0.707207  1 H  s                  6      0.707207  2 H  s          
     3     -0.434716  1 H  px                 8     -0.434716  2 H  px         

 Vector    5  Occ=0.000000D+00  E= 1.441413D+00
              MO Center= -1.1D-16, -3.1D-18,  9.8D-18, r^2= 4.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.533228  1 H  pz                10      0.533228  2 H  pz         
     4     -0.169016  1 H  py                 9     -0.169016  2 H  py         

 Vector    6  Occ=0.000000D+00  E= 1.441413D+00
              MO Center= -6.4D-17, -3.3D-17, -1.0D-17, r^2= 4.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.533228  1 H  py                 9      0.533228  2 H  py         
     5      0.169016  1 H  pz                10      0.169016  2 H  pz         

 Vector    7  Occ=0.000000D+00  E= 2.238214D+00
              MO Center=  1.1D-16,  2.1D-17, -3.1D-17, r^2= 7.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.806254  1 H  px                 8     -0.806254  2 H  px         
     1      0.438679  1 H  s                  6      0.438679  2 H  s          
     2     -0.221215  1 H  s                  7     -0.221215  2 H  s          

 Vector    8  Occ=0.000000D+00  E= 2.330384D+00
              MO Center=  4.2D-16,  9.0D-20, -6.8D-20, r^2= 6.6D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.891280  1 H  py                 9     -0.891280  2 H  py         
     5     -0.670278  1 H  pz                10      0.670278  2 H  pz         

 Vector    9  Occ=0.000000D+00  E= 2.330384D+00
              MO Center=  1.7D-16, -6.7D-20, -8.9D-20, r^2= 6.6D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.891280  1 H  pz                10     -0.891280  2 H  pz         
     4      0.670278  1 H  py                 9     -0.670278  2 H  py         

 Vector   10  Occ=0.000000D+00  E= 4.963601D+00
              MO Center= -2.2D-16,  3.2D-22, -2.1D-21, r^2= 6.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      4.336130  1 H  s                  6     -4.336130  2 H  s          
     3     -2.373918  1 H  px                 8     -2.373918  2 H  px         
     2     -0.580176  1 H  s                  7      0.580176  2 H  s          


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           0.000000000000           0.000000000000           0.000000000000
           0.000000000000           0.647821420228           0.000000000000
           0.000000000000           0.000000000000           0.647821420228

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 H    1     1.00   0.76  0.23  0.01
    2 H    1     1.00   0.76  0.23  0.01

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0     -0.000000      0.000000      2.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0     -0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000

     2   2 0 0     -1.017579      0.000000      0.642792
     2   1 1 0     -0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -1.346441      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -1.346441      0.000000      0.000000


 Parallel integral file used      16 records with       0 large values


 Task  times  cpu:        0.1s     wall:        0.2s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  120      120      343      176      237        0        0        5     
number of processes/call 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes total:             1.27e+05 7.65e+04 1.56e+04 0.00e+00 0.00e+00 4.00e+01
bytes remote:            2.62e+03 2.49e+03 2.80e+02 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 8160 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        17	        14
	current total bytes		         0	         0
	maximum total bytes		   1053752	  22509176
	maximum total K-bytes		      1054	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

          E. Aprà, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, Y. Alexeev, J. Anchell,
       V. Anisimov, F. W. Aquino, R. Atta-Fynn, J. Autschbach, N. P. Bauman,
     J. C. Becca, D. E. Bernholdt, K. Bhaskaran-Nair, S. Bogatko, P. Borowski,
        J. Boschen, J. Brabec, A. Bruner, E. Cauët, Y. Chen, G. N. Chuev,
      C. J. Cramer, J. Daily, M. J. O. Deegan, T. H. Dunning Jr., M. Dupuis,
   K. G. Dyall, G. I. Fann, S. A. Fischer, A. Fonari, H. Früchtl, L. Gagliardi,
      J. Garza, N. Gawande, S. Ghosh, K. Glaesemann, A. W. Götz, J. Hammond,
       V. Helms, E. D. Hermes, K. Hirao, S. Hirata, M. Jacquelin, L. Jensen,
   B. G. Johnson, H. Jónsson, R. A. Kendall, M. Klemm, R. Kobayashi, V. Konkov,
      S. Krishnamoorthy, M. Krishnan, Z. Lin, R. D. Lins, R. J. Littlefield,
      A. J. Logsdail, K. Lopata, W. Ma, A. V. Marenich, J. Martin del Campo,
   D. Mejia-Rodriguez, J. E. Moore, J. M. Mullin, T. Nakajima, D. R. Nascimento,
    J. A. Nichols, P. J. Nichols, J. Nieplocha, A. Otero-de-la-Roza, B. Palmer,
    A. Panyala, T. Pirojsirikul, B. Peng, R. Peverati, J. Pittner, L. Pollack,
   R. M. Richard, P. Sadayappan, G. C. Schatz, W. A. Shelton, D. W. Silverstein,
   D. M. A. Smith, T. A. Soares, D. Song, M. Swart, H. L. Taylor, G. S. Thomas,
            V. Tipparaju, D. G. Truhlar, K. Tsemekhman, T. Van Voorhis,
     Á. Vázquez-Mayagoitia, P. Verma, O. Villa, A. Vishnu, K. D. Vogiatzis,
        D. Wang, J. H. Weare, M. J. Williamson, T. L. Windus, K. Woliński,
        A. T. Wong, Q. Wu, C. Yang, Q. Yu, M. Zacharias, Z. Zhang, Y. Zhao,
                                and R. J. Harrison
                        "NWChem: Past, present, and future
                         J. Chem. Phys. 152, 184102 (2020)
                               doi:10.1063/5.0004997

                                      AUTHORS
                                      -------
     E. Apra, E. J. Bylaska, N. Govind, K. Kowalski, M. Valiev, W. A. de Jong,
      T. P. Straatsma, H. J. J. van Dam, D. Wang, T. L. Windus, N. P. Bauman,
       A. Panyala, J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec,
    K. Lopata, S. A. Fischer, S. Krishnamoorthy, M. Jacquelin, W. Ma, M. Klemm,
       O. Villa, Y. Chen, V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler,
           Eric Hermes, L. Jensen, J. E. Moore, J. C. Becca, V. Konjkov,
            D. Mejia-Rodriguez, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
        P.-D. Fan, A. Fonari, M. J. Williamson, R. J. Harrison, J. R. Rehr,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
      M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
     G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols,
       K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski,
       T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood,
      E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju,
        R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        0.1s     wall:        0.3s
[0] ARMCI Warning: Freed 1 leaked allocations
